renderer/qgl.h

```diff
@@ -24,6 +24,7 @@
 #define GL_KHR_debug							1	//core since 4.3
 #define GL_ARB_stencil_texturing				1	//core since 4.3
 #define GL_ARB_buffer_storage					1	//core since 4.4
+#define GL_ARB_texture_storage					1	//core since 4.2
```

Comment:
Please add CHECK_FEATURE call in qgl.cpp.
It is confusing when game uses many extensions, but does not report them to console.

Reply:
Done.